index.js

```diff
@@ -0,0 +1,217 @@
+}
+
+//       format :: BenchmarkResult -> String
+function format(res) {
+  return res.hz.toLocaleString('en', {maximumFractionDigits: 0}) + ' Hz '
+       + '±' + res.stats.rme.toFixed(2) + '% '
+       + '(n ' + String(res.stats.sample.length) + ')';
+}
+
+//       get :: (String, a, Any) -> a
+function get(key, fallback, x) {
+  var o = Object(x);
+  return typeof o[key] === typeof fallback ? o[key] : fallback;
+}
+
+//       green :: String -> String
+function green(s) {
+  return '\u001B[32m' + s + '\u001B[0m';
+}
+
+//       identity :: a -> a
+function identity(x) {
+  return x;
+}
+
+//       red :: String -> String
+function red(s) {
+  return '\u001B[31m' + s + '\u001B[0m';
+}
+
+//.
+//. ## API Documentation
+//.
+//. <details><summary>
+//. <code>default :: (a, b, StrMap (Spec a b)) -> Options -> Undefined</code>
+//. </summary>
+//.
+//. ```hs
+//. Spec a b :: ((a | b) -> Undefined)
+//.           | { left :: a -> Undefined, right :: b -> Undefined }
+//.
+//. Options :: { colors :: Boolean?
+//.              leftHeader :: String?
+//.              match :: String?
+//.              minSamples :: Number?
+//.              rightHeader :: String?
+//.              significantChange :: Number? }
+//.
+//. default :: (a, b, StrMap (Spec a b)) -> Options -> Undefined
+//. ```
+//.
+//. </details>
+//.
+//. This module exports a single function. It takes three arguments and returns
+//. another function. The three arguments are:
+//.
+//. 1. The left-hand benchmarking input: could be an older version of the
+//.    library you're testing, or a competing library.
+//. 2. The right-hand benchmarking input: usually the current version of the
+//.    library you're testing `require`'d directly from the working directory.
+//. 3. A mapping of benchmarks where the keys represent the names, and the
+//.    values describe the work we're benchmarking. The names can later be used
+//.    to filter benchmarks by using a glob, so it's recommended to use the
+//.    forward slash character as a separator, as shown in [usage](#usage).
+//.    The value should be a function which will be called with both the
+//.    left-hand benchmarking input (`1`) *and* the right-hand benchmarking
+//.    input (`2`). Alternatively when the two inputs are not the same, an
+//.    object with a `left` and `right` property, each containing a function,
+//.    may be provided.
+//.
+//. Once these inputs are provided, a function is returned. The function will
+//. run the benchmarks and print the results to StdOut when it is called. It
+//. takes as input an object of options for the customization of this process:
+//.
+//. * `colors` (`true`): Set to `false` to disable terminal colors.
+//. * `leftHeader` (`"left"`): This String will be printed in the table above
+//.   the left-hand benchmarking results.
+//. * `match` (`"**"`): This glob allows one to filter benchmarks by name.
+//. * `minSamples` (`80`): The minimum number of benchmarking samples to take.
```

Comment:
Because it was the number used in the code originally written by @safareli :man_shrugging: